tests/users/tests_register_api.py

```diff
@@ -4,8 +4,101 @@
 
 class RegistrationTestCase(APITestCase):
 
-    def tests_registration(self):
-        data = {"email" : "test@test.test","name" : "test","institution": "testing","password": "testingg","country_code": "+00","phone": "0000000000"}
+
+    def test_registration_success_with_correct_credentials(self):
+        data = {
+            "email" : "test@test.test",
+            "name" : "test",
+            "institution": "testing",
+            "password": "testingg",
+            "country_code": "+00",
+            "phone": "0000000000"
+        }
+
+        response = self.client.post("/api/users/register/", data)
+        self.assertEqual(response.status_code , status.HTTP_201_CREATED)
+        self.assertNotEqual(response.status_code , status.HTTP_400_BAD_REQUEST)
```

Comment:
Umm yeahh, since there were only two response codes I thought it would be nice to include them both in the test cases, I could delete the NotEqual lines from the cases too. Do I ?

Reply:
Yup Please do so :innocent:

Reply:
Alright sir, did that!